### PR TITLE
chore(mysqldump): support mysqldump dump schema 

### PR DESCRIPTION
**Pull request by BohuTANG**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Reference MySQL 8.0:
https://github.com/mysql/mysql-server/blob/8.0/client/mysqldump.cc

Export:
```
mysqldump --set-gtid-purged=OFF --no-tablespaces --no-data --triggers=0 --column-statistics=0 -uroot -h127.0.0.1 -P3307 dd1 >/tmp/dd1.sql
```

```
-- MySQL dump 10.13  Distrib 8.0.28, for Linux (x86_64)
--
-- Host: 127.0.0.1    Database: dd1
-- ------------------------------------------------------
-- Server version	8.0.26-v0.7.23-nightly-0f576a7-simd(rust-1.61.0-nightly-2022-04-20T08:39:39.983444754Z)

/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
/*!50503 SET NAMES utf8mb4 */;
/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
/*!40103 SET TIME_ZONE='+00:00' */;
/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;

--
-- Table structure for table `t1`
--

DROP TABLE IF EXISTS `t1`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `t1` (
  `a` INT,
) ENGINE=FUSE;
/*!40101 SET character_set_client = @saved_cs_client */;


/*!40101 SET SQL_MODE=@OLD_SQL_MODE */;
/*!40014 SET FOREIGN_KEY_CHECKS=@OLD_FOREIGN_KEY_CHECKS */;
/*!40014 SET UNIQUE_CHECKS=@OLD_UNIQUE_CHECKS */;
/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
/*!40111 SET SQL_NOTES=@OLD_SQL_NOTES */;

-- Dump completed on 2022-04-20 19:37:26

```

Import:
```
mysql -h127.0.0.1 -uroot -P3307 dd1 < /tmp/dd1.sql
```

## Changelog

- New Feature

## Related Issues

Fixes #issue

### CODE REVIEW

**Comment by TCeason** [comment]
LGTM

File: query/src/sql/sql_parser.rs [hunk 1]
```diff
@@ -346,6 +346,8 @@ impl<'a> DfParser<'a> {
             Ok(DfStatement::ShowSettings(DfShowSettings))
         } else if self.consume_token("CREATE") {
             self.parse_show_create()
+        } else if self.consume_token("FIELDS") {
```
Comment (TCeason) [inline query/src/sql/sql_parser.rs:349]: > } else if self.consume_token("FIELDS") {

} else if self.consume_token("FIELDS") || self.consume_token("COLUMNS") {

maybe we can also support `columns`?
```SQL
SHOW [EXTENDED] [FULL] {COLUMNS | FIELDS}
    {FROM | IN} tbl_name
    [{FROM | IN} db_name]
    [LIKE 'pattern' | WHERE expr]
```

Comment (BohuTANG) [reply query/src/sql/sql_parser.rs:349]: Good idea, will try it later in this PR

Comment (BohuTANG) [reply query/src/sql/sql_parser.rs:349]: Oh, I have a check, this syntax is not support LIKE and only for mysqldump not for user in databend now.

**Pull request merged**